src/api/app/lib/backend/api/build_results/binaries.rb

```diff
@@ -47,8 +47,8 @@ def self.rpmlint_log(project_name, package_name, repository_name, architecture_n
 
         # special view on a binary file for details display
         # @return [Hash]
-        def self.fileinfo_ext(project_name, package_name, repository, arch, filename)
-          fileinfo = http_get(['/build/:project/:repository/:arch/:package/:filename?view=fileinfo_ext', project_name, repository, arch, package_name, filename])
+        def self.fileinfo_ext(project_name, package_name, repository, arch, filename, filelist=0)
```

Comment:
`self.fileinfo_ext` could be modified to accept `options` as last parameter, instead of `filelist`. You can take this other method as inspiration: https://github.com/openSUSE/open-build-service/blob/33c5a02429dd80905608397759aadbef265a5b63/src/api/app/lib/backend/api/sources/package.rb#L27-L32

I will also make clearer what parameter are we setting to `1` or `0` in the calls. And there is also an opportunity to refactor the `view` query parameter of the line below.